templates/nextjs/test/contract.test.ts

```diff
@@ -0,0 +1,59 @@
+import { launchTestNode } from 'fuels/test-utils';
+
+import { describe, test, expect } from 'vitest';
+
+/**
+ * Imports for the contract factory and bytecode, so that we can use them in the test.
+ *
+ * Can't find these imports? Make sure you've run `fuels build` to generate these with typegen.
+ */
+import { TestContractAbi__factory } from '../src/sway-api';
+import bytecode from '../src/sway-api/contracts/TestContractAbi.hex';
+
+/**
+ * @group node
```

Comment:
```suggestion
 * @group browser
 * @group node
```

We can run these in browser can't we? (Haven't added to the others)

Reply:
I think we shouldn't have these `@group` comments in the templates. They are related to the way _we_ run our test suites and we shouldn't be porting that to the end user. Rather, wherever we're running tests related to the `create-fuels` utility, that's where we should run the `pnpm test` command suggested in https://github.com/FuelLabs/fuels-ts/pull/2755#discussion_r1674172931 and verify the tests are working properly in that way.

Reply:
Added the browser group and opted to [strip the groups](https://github.com/FuelLabs/fuels-ts/pull/2755/commits/04dd9d5f45ded8caf410597ee7481371e5985cca) from the template like we do for other template files.